doc/dependencies.rst

```diff
@@ -12,7 +12,7 @@ Security aspects
 ----------------
 
 A project that uses Hunter first downloads the hunter package itself. This is
```

Comment:
* Can you please capitalize the second instance of Hunter? 
* Can you remove "itself"?

Sentence should read: 

"A project that uses Hunter first downloads the Hunter package."

Reply:
This sounds a bit ambiguous, "hunter package" usually refers to some library packaged for Hunter, while this sentence I think refers to downloading some core Hunter stuff. Any ideas how to better reformulate?

Reply:
Perhaps "Hunter client"? The term is used here: https://docs.hunter.sh/en/latest/